### PR TITLE
k8smeetup-tools-index-pr-2017-09-17

### PR DESCRIPTION
**Pull request by Tony-CCIE**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5507)

### CODE REVIEW

File: cn/docs/tools/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#### 本地工具
+
+kubernetes中包含了以下内建工具
+
+
+
+##### kubectl
+
+[`kubectl`](/docs/user-guide/kubectl/)是kubernetes的命令行工具。它控制着kubernetes的集群管理。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tools/index.md]: 它控制着kubernetes的集群管理。
->
它可以被用于管理Kubernetes集群。

File: cn/docs/tools/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+##### kubeadm
+
+[`kubeadm`](/docs/getting-started-guides/kubeadm/)可以在物理、云服务器或是虚拟机(目前处于alpha）上，轻松配置安全的kubernetes集群的命令行工具。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tools/index.md]: 物理
->
物理机

File: cn/docs/tools/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+##### kubefed
+
+[`kubefed`](/docs/tutorials/federation/set-up-cluster-federation-kubefed/)是帮助您管理您的联邦集群的命令行工具。
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tools/index.md]: 是帮助您管理您的联邦集群的命令行工具。
->
是帮助您管理联邦集群的命令行工具。

File: cn/docs/tools/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+##### Dashboard
+
+[Dashboard](/docs/tasks/web-ui-dashboard/),是基于web用户界面的kubernetes,允许您部署容器应用到kubernetes集群中，并对其进行故障排除，以及管理集群和其自身的资源。
+
+
+
+#### 第三方工具
+
+kubernetes支持有效的第三方工具。包括这些，但不限制于这些：
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tools/index.md]: 包括这些，但不限制于这些：
->
包括但不限于以下工具：

**Comment by steveperry-53** [comment]
@Please rebase your branch so that it picks up 99fbc2b. That will fix the deploy/netlify failure. Thanks.

**Comment by zhangxiaoyu-zidif** [comment]
@Tony-CCIE 
please rebase your codes, thanks=)

**Pull request closed without merging**